xray/audit/java/gradle.go

```diff
@@ -40,10 +43,29 @@ func runGradle(buildConfiguration *utils.BuildConfiguration, excludeTestDeps, us
 			log.Debug("Using resolver config from", configFilePath)
 		}
 	}
+	// check if gradle wrapper exist
+	if useWrapper {
+		var wrapperExist bool
+		wrapperExist, err = isGradleWrapperExist()
+		if err != nil {
+			return
+		}
+		useWrapper = wrapperExist
+	}
 	// Read config
 	vConfig, err := utils.ReadGradleConfig(configFilePath, useWrapper)
 	if err != nil {
 		return err
 	}
 	return gradleutils.RunGradle(vConfig, tasks, "", buildConfiguration, 0, useWrapper, true)
 }
+
+// This function assumes that the Gradle wrapper is in the root directory.
+// Adapting this function is needed if the audit command supports Gradle's --project-dir option.
```

Comment:
"Adapting this function is needed" - not sure what this means. Which adaptation are you referring to?